### PR TITLE
Added more author hails.

### PR DESCRIPTION
**Pull request by pilover100**
## Summary
Added more author hails to better inform people about the ES community discord server.

## Save File
Any will work, just hail a author ship that has the friendly author hails such as "Power of the People" to see the new hails.

## PR Checklist
 - [X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [ ] I uploaded the necessary image, blend, and texture assets here: {{insert link to assets}}
 - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}
 
Just text changes, I decline copyright of the textual changes and no images where altered or added.

### CODE REVIEW

File: data/human/hails.txt [hunk 1]
```diff
@@ -2779,33 +2779,37 @@ phrase "friendly pirate"
...
+		"contributing ideas for improvements."
+		"contributing landscape photos."
+		"contributing artwork."
+		"contributing new ships of your own design."
+		"contributing plugins."
+		"contributing new special missions."
+		"contributing new weapons or other outfits."
+		"contributing improved planet descriptions."
+		"contributing more varied hail messages."
+		"joining conversations creating new content."
```
Comment (roadrunner56) [inline data/human/hails.txt:2809]:
```suggestion
		"joining conversations that create new content."
```

File: data/human/hails.txt [hunk 2]
```diff
@@ -2779,33 +2779,37 @@ phrase "friendly pirate"
...
+		"joining the Endless Sky community discord."
```
Comment (roadrunner56) [inline data/human/hails.txt:2810]:
```suggestion
		"joining the Endless Sky Community Discord."
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

**Review by roadrunner56 — APPROVED** [review]
Looks good!

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -19,6 +19,7 @@ on:
     # Run for pushes to master or a version branch, e.g. a PR was merged ...
     branches:
       - master
+      - test
```
Comment (Amazinite) [inline .github/workflows/ci.yml:22]: Unrelated change.

**Review by Amazinite — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 3 above)
Comment (Amazinite) [reply .github/workflows/ci.yml:22]:
```suggestion
```

Comment (pilover100) [reply .github/workflows/ci.yml:22]: Ah... that bit, wasn't intended for this PR, mostly just for testingstuff my end. How should I sort this out?

Comment (Amazinite) [reply .github/workflows/ci.yml:22]: Delete the line.

**Comment by pilover100** [comment]
Thanks Amazinite, I'll make sure to tick the allow edit box next time to make your life easier. Sorry about that!
